README.md

```diff
@@ -9,12 +9,42 @@ A Foreman smart proxy plugin for Katello.  Implements container registry functio
    SSLRequire %{SSL_CLIENT_S_DN_CN} eq "admin"
 </Location>
 
+SSLProxyCheckPeerCN off
+SSLProxyCheckPeerName off
+
 ProxyPass /pulpcore_registry/v2/ http://127.0.0.1:24817/v2/
 ProxyPassReverse /pulpcore_registry/v2/ http://127.0.0.1:24817/v2/
 
-ProxyPass /pulp/container http://127.0.0.1:24816/pulp/container
-ProxyPassReverse http://127.0.0.1:24816/pulp/container  /pulp/container
+ProxyPass /pulp/container/ unix:///run/pulpcore-content.sock|http://centos7-katello-devel.cannolo.example.com/pulp/container/
+ProxyPassReverse /pulp/container/ unix:///run/pulpcore-content.sock|http://centos7-katello-devel.cannolo.example.com/pulp/container/
 
 ProxyPass /v2 https://127.0.0.1:9090/container_gateway/v2
 ProxyPassReverse https://127.0.0.1:9090/container_gateway/v2 /v2
 ```
+
+# Server configuration
+
+The Container Gateway plugin requires a Pulp 3 instance to connect to.  Related configuration options:
+```
+:pulp_endpoint: 'https://your_pulp_3_server_here.com'
+:pulp_client_ssl_cert: 'Path to X509 certificate for authenticating with Pulp'
```

Comment:
The current foreman-installer doesn't properly set ownership or permissions, so manual adjustments were required to get these keys readable by the plugin.